test/features/parsoid/transform.js

```diff
@@ -15,197 +15,167 @@ describe('transform api', function() {
     this.timeout(20000);
     let contentTypes;
     const server = new Server();
+    const client = new REST('');
     before(() => {
         return server.start()
         .then(() => {
             contentTypes = server.config.conf.test.content_types;
-            return preq.get({
-                uri: `${server.config.bucketURL('en.wikipedia.beta.wmflabs.org')}/html/${testPage.title}/${testPage.revision}`
-            });
+            return client.get(
+                `${server.config.bucketURL('en.wikipedia.beta.wmflabs.org')}/html/${testPage.title}/${testPage.revision}`
+            );
         })
         .then((res) => {
-            testPage.html = res.body;
+            testPage.html = res.text;
         });
     });
     after(() => server.stop());
 
     it('wt2html', () => {
-        return preq.post({
-            uri: `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/wikitext/to/html/${testPage.title}`,
-            body: {
-                wikitext: '== Heading =='
-            }
-        })
+        return client.post(
+            `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/wikitext/to/html/${testPage.title}`,
+            { wikitext: '== Heading ==' }
+        )
         .then((res) => {
             assert.deepEqual(res.status, 200);
-            assert.contentType(res, contentTypes.html);
+            assert.contentTypes(res, mwUtil.constructRegex([contentTypes.html]));
             const pattern = /<h2.*>Heading<\/h2>/;
```

Comment:
can you replace all pattern matchings with assert.match?

Reply:
Thanks @Pchelolo all pattern matchings are now using `assert.match` as per your advice.